src/org/gk/gkCurator/CuratorActionCollection.java

```diff
@@ -2169,10 +2169,41 @@ private void deleteInstance() {
 		}
 	}
 	
+	/**
+	 * Retrieve a "Create Instance" action for new entities.
+	 * 
+	 * @return Action with specific "SHORT_DESCRIPTION" and "enabled" status
+	 */
 	public Action getCreateInstanceAction() {
-		if (createInstanceAction == null) {
+		// list of constants that curators are not allowed to create new instances of.
+		ArrayList<String> nonCreatable = new ArrayList<String>();
```

Comment:
Could the declaration be for `List<String>`?

Reply:
Absolutely! I will update.

Reply:
Great, thanks!